shared/tsconfig.json

```diff
@@ -9,6 +9,7 @@
         "tag": "gql",
       },
     ],
+    "downlevelIteration": true,
```

Comment:
Why is this needed? We only target modern browsers so generally shouldn't do this as it could bloat bundles. Is this significant to fix the bug?

Reply:
This is the only thing that fixes the bug. The double spread is just a drive-by.